_extras/guide.md

```diff
@@ -167,6 +167,26 @@ particular set of files in `.gitignore`.
     repositories in the web and provides a web interface to interact with repos
     they host.
 
+*   Using personal access tokens to authenticate with GitHub from the command line
+    might be confusing for some learners as GitHub has only recently deprecated
+    password authentication. You can point advanced learners interested in the
+    rationale behind the decision by GitHub to [this blog post explaining it][github-access-tokens].
+
+*   Doing a live demo of how a personal access token is generated has the intrinsic
+    disadvantage that the learners can see the token and can therefore get access
+    to your GitHub account by using it. There is a few good strategies to address this
+    issue though:
+
+    *   Immediately revoke the token that you generated after you showed everybody
+        how generating the token works. For the rest of the course, work with a token
+        that you have generated without learners watching (e.g. before the course)
+
+    *   Be even more prudent about this in online teaching, where learners are able
+        to take screenshots.
+
```

Comment:
I believe a "safer" suggestion would be to use a new account or one only for teaching. That's what I normally do so I know I'm seeing something similar to what the learners are seeing and it doesn't contain any information to other projects/notifications I may be working on.  Certainly, we should follow the practices you suggested here, but in case I forget to revoke that token I'm not in big trouble.